app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -537,6 +707,93 @@ const CompanyInterestPage = (props: Props) => {
           component={TextInput.Field}
           required
         />
+        <Flex column className={styles.interestBox}>
+          <label htmlFor="companyType" className={styles.heading}>
+            {labels.companyTypes[language]}
+          </label>
+          <RadioButtonGroup name="companyType">
+            {Object.keys(COMPANY_TYPES).map((key, index) => (
+              <Field
+                key={key}
+                name={key}
+                label={COMPANY_TYPES[key][language]}
+                component={RadioButton.Field}
+                inputValue={key}
+              />
+            ))}
+          </RadioButtonGroup>
+        </Flex>
+        <Flex column className={styles.interestBox}>
+          <label htmlFor="accomodating" className={styles.heading}>
+            {labels.accomodating[language]}
+          </label>
+          <RadioButtonGroup name="accomodating">
+            {Object.keys(ACCOMODATING).map((key, index) => (
+              <Field
+                key={key}
+                name={key}
+                label={ACCOMODATING[key][language]}
+                component={RadioButton.Field}
+                inputValue={key}
+              />
+            ))}
+          </RadioButtonGroup>
+        </Flex>
+        <Flex column className={styles.interestBox}>
+          <label htmlFor="companySurveyOffers" className={styles.heading}>
+            <Flex alignItems="center" gap="5px">
+              {labels.companySurveyOffers[language]}
+              <Tooltip
+                className={styles.tooltip}
+                renderDirection="right"
+                content={
+                  <span>
+                    {language === 'norwegian'
+                      ? 'Dette er temaer fra bedriftsundersøkelsen som studenter uttrykte interesse for å lære mer om. Vil noen av disse være av interesse å inkludere for dere? (Uforpliktende)'
+                      : 'These are topics from the company survey that students expressed interest in learning more about. Would any of these be of interest to you to include? (Non-binding)'}
+                  </span>
+                }
+              >
+                <Icon name="alert-circle-outline" />
+              </Tooltip>
+            </Flex>
+          </label>
+
+          <FieldArray
+            label="companySurveyOffers"
+            name="companySurveyOffers"
+            language={language}
+            component={SurveyOffersBox}
+          />
+        </Flex>
+        <div className={styles.topline}>
+          {interestText.text.first[language]}
+          {/*
+          <br />
+          <br />
+          {interestText.text.second[language]}
+          <br />
+          <br />
+          {interestText.bedex[language]
+          <br />
+          <br />
+          {interestText.anniversaryCollaboration[language]}
+          */}
+          {/*           <br />
+          <br />
+          {interestText.revueCollaboration[language]}
+ */}{' '}
```

Comment:
?

Reply:
We got a request to remove the revue collab box ^